core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -105,14 +83,14 @@
    */
   public static final String FIELD_COUNT_PROPERTY_DEFAULT = "10";
 
-  int fieldcount;
+  private int fieldcount;
```

Comment:
The other thought here is to actually get the property value yourself instead of relying on CoreWorkload to populate the variable.

```java
String table = p.getProperty(CoreWorkload.TABLENAME_PROPERTY, CoreWorkload.TABLENAME_PROPERTY_DEFAULT);
```